storage/backend/ceph/ceph.go

```diff
@@ -82,16 +82,20 @@ func (c *Driver) unlockAndLog(do storage.DriverOptions) {
 // Format formats a created volume.
 func (c *Driver) Format(do storage.DriverOptions) error {
 	device, err := c.mapImage(do)
-	if err != nil {
+	if err == ErrLockFailed {
```

Comment:
looks like we try to unlock as part of error handling so wondering, if this check should be `err != ErrLockFailed`?

I am also wondering why can't we unlock always whenever `err != nil`, that way we can just `defer` the call to unlock when err is not nil?